Fixed the errors and the test results are shown in the README.md file

ace-coder opened the pull request.

ace-coder committed:
Fixed the errors and the test results are shown in the README.md file

domthewop commented:
Unnecessary changes to SQL statements. Changed unit test files. Pushed vendor libraries to repository.

ace-coder commented:
Hi,

Sorry for the last commit.

Please check the new pull request.

   1.

   I did not change the unit test files. There was a space in a test file,
   now I removed that.
   2.

   In the testQuestion test we have assertNotEmpty check but the db has
   empty values for _answer_ fields which should not happen. In the db
   script those fields are declared as NOT NULL but those are empty fields.
   That's why I added text to those fields to make sure it's realistic.
   3.

   I have removed the vendor library files.

I can assure you that you will never be unhappy with my work. I have great
skills on PHP/Wordpress and front-end stuff.

Best regards,
Mainuddin

On Mon, Sep 29, 2014 at 12:51 AM, edgeprod notifications@github.com wrote:

> Unnecessary changes to SQL statements. Changed unit test files. Pushed
> vendor libraries to repository.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/edgeprod/PHP-assessment/pull/7#issuecomment-57095671.
